app/src/main/java/app/revanced/manager/ui/viewmodel/InstallerViewModel.kt

```diff
@@ -147,24 +133,57 @@ class InstallerViewModel(
                                 input.selectedPatches
                             )
                         }
-                    } else {
-                        app.toast(app.getString(R.string.install_app_fail, extra))
                     }
+
+                    packageInstallerResult = PackageInstallerResult(pmStatus, pmExtra)
+                    showInstallerStatusDialog = true
                 }
 
                 UninstallService.APP_UNINSTALL_ACTION -> {
+                    val pmStatus = intent.getIntExtra(
+                        UninstallService.EXTRA_UNINSTALL_STATUS,
+                        PackageInstaller.STATUS_FAILURE
+                    )
+                    val pmExtra =
+                        intent.getStringExtra(UninstallService.EXTRA_UNINSTALL_STATUS_MESSAGE)
+
+                    packageInstallerResult = PackageInstallerResult(
+                        pmStatus,
+                        pmExtra,
+                        uninstall = true
+                    )
+                    showInstallerStatusDialog = true
                 }
             }
         }
     }
 
     init {
-        ContextCompat.registerReceiver(app, installBroadcastReceiver, IntentFilter().apply {
-            addAction(InstallService.APP_INSTALL_ACTION)
-            addAction(UninstallService.APP_UNINSTALL_ACTION)
-        }, ContextCompat.RECEIVER_NOT_EXPORTED)
+        // TODO: navigate away when system-initiated process death is detected
+        //  because it is not possible to recover from it.
```

Comment:
Move this to the `launchExpedited` call site

Reply:
I am not sure what `launchExpedited` is. This LoC is not part of the PR.

Reply:
The comment was above an init block before this PR, but code was moved around so the comment is in a weird place.

It would be best to have it above the call to `workerRepository.launchExpedited`, which is the function that launches the patcher worker. There might be another place for the comment, but the current location is weird and should be changed.

Reply:
Marking as resolved as this exceeds my field of expertise and is off-topic for this PR

Reply:
It is on-topic because this PR moves and removes a lot of code in the file, but the comment wasn't moved to account for it.

Just move [here](https://github.com/ReVanced/revanced-manager/blob/63f2e2f167d0800db4408c470a35cddbf1e7807e/app/src/main/java/app/revanced/manager/ui/viewmodel/InstallerViewModel.kt#L101)

Reply:
I am not sure what to do here. What do you mean by "Just move here"

Reply:
Take the comment marked by the review and move it to the specified location